osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/configuration/service/ProtocolService.java

```diff
@@ -0,0 +1,11 @@
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.configuration.service;
```

Comment:
ProtocolService is a marker interface which can be replaced by naming the @Component.

Reply:
ProtolService is not a marker interface. It declares `boolean handles(Protocol)` which allows services to determine which protocols they can handle.